### PR TITLE
Make code snippets from docstring

### PR DESCRIPTION
**Pull request by nikita-0209**
### Description

This pull request is to make code snippets from docstrings in class `Angle`

Fixes #<Issue Number>

### CODE REVIEW

File: astropy/coordinates/angles.py [hunk 1]
```diff
@@ -33,24 +33,24 @@ class Angle(u.SpecificTypeQuantity):
...
-      Angle('-1h2m3s')
-      Angle('-1h2m3sE')
-      Angle('-1h2.5m')
-      Angle('-1h2.5mW')
-      Angle('-1:2.5', unit=u.deg)
-      Angle((10, 11, 12), unit='hourangle')  # (h, m, s)
-      Angle((-1, 2, 3), unit=u.deg)  # (d, m, s)
-      Angle(10.2345 * u.deg)
-      Angle(Angle(10.2345 * u.deg))
+      >>> Angle('10.2345d')
```
Comment (bsipocz) [inline astropy/coordinates/angles.py:36]: these doctested examples now need to be full examples, with appropriate imports, etc.

Comment (nikita-0209) [reply astropy/coordinates/angles.py:36]: do you mean that the first shown example needs to be preceded by the imports required to declare an instance of this class?

Comment (bsipocz) [reply astropy/coordinates/angles.py:36]: Yes. If you copy paste these  as-is into an interpreter it should be able to run without issues.

**Review by embray — APPROVED** [review]
I addressed the pep8 nitpicks.  Approved, pending successful CI re-run.  Thanks @nikita-0209

**Comment by eteq** [comment]
The CI failures are all read herrings, and I got this to run locally without errors so I'm merging to clean out v4.1 issues.

**Comment by astrojuanlu** [comment]
xref: #9485

**Pull request merged**